newyork.html

```diff
@@ -290,7 +374,7 @@ <h3>A Special Time</h3>
                     <div class="news-grid1">
```

Comment:
## Styling  Improvement?
- [ ] Would you mind improving the styling on this section to appear as present on the original page. The elements ought to be compact together in the middle.
- [ ] There is also a section that is missing. The `read 615 comments` and the social links right after the main article. 

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html#commentsContainer)|[Clone Page](https://raw.githack.com/Amalcxc/New-York-Times/one-page-article/newyork.html)|
|:--:|:--:|
|![sytle-imporment-suggestion](https://user-images.githubusercontent.com/22345961/108572570-a9844680-7323-11eb-9a7e-3e8a079fd0b0.png)|![sytle-imporment-suggestion-clone](https://user-images.githubusercontent.com/22345961/108572585-b012be00-7323-11eb-8311-cc0709f19906.png)|